### PR TITLE
NEW - Add param color button action

### PR DESCRIPTION
**Pull request by BB2A-Anthony**
# NEW|New [*Add param color button action*]

### CODE REVIEW

File: htdocs/theme/eldy/global.inc.php [hunk 1]
```diff
@@ -358,7 +358,7 @@
 	padding-right: 5px;
 	background-color: transparent;
 	color: var(--colortext) !important;
-	border: 2px solid var( --butactionbg) !important;
+	border: 2px solid rgb(<?php echo $butactionbg ?>) !important;
```
Comment (eldy) [inline htdocs/theme/eldy/global.inc.php:361]: You must keep the var( --butactionbg)   (everywhere)
and add only at one place the <?php echo $butactionbg ?> where the var  butactionbg is defined.

Comment (atm-john) [reply htdocs/theme/eldy/global.inc.php]: take a look at this https://www.youtube.com/watch?v=IHaT_rjC2gM this guy explain a good usage of css var

Comment (bb2a) [reply htdocs/theme/eldy/global.inc.php]: @atm-john @eldy Sory ok for me i change

it's done

**Comment by bb2a** [comment]
@eldy ok for you? I resolved conflict.

**Pull request closed without merging**